src/vis_cpu/conversions.py

```diff
@@ -258,7 +267,60 @@ def equatorial_to_eci_coords(ra, dec, obstime, location, unit="rad", frame="icrs
     return pra, pdec
 
 
-def uvbeam_to_lm(uvbeam, freqs, n_pix_lm=63, polarized=False, **kwargs):
+def prepare_beam(
+    uvbeam: UVBeam, polarized: bool = False, use_feed: Literal["x", "y"] = "x"
```

Comment:
do you want to allow for "n" or "e" if `x_orientation` is set on the UVBeam object?

Reply:
Not right now.